system/services/admin/DataManagerService.cfc

```diff
@@ -227,6 +238,47 @@ component output="false" singleton=true {
 
 		return result;
 	}
+	public struct function batchSaveFieldChanges(
+		  required string  objectName
+		, required string  sourceIds
+		, required string  updateValue
+		, required string  updateField
+		, required string  DataColumn
```

Comment:
should be "headless" camel case, i.e. `dataColumn` _not_ `DataColumn`. Also, intention of this variable is not clear in the name; `forekgnKeyField` may be clearer